docs/src/frontend/index.md

```diff
@@ -696,6 +696,26 @@ acc.tokensAccepted() => Promise<Array<Token>>
 Returns a Promise for an array of tokens that are accepted by `acc`.
 On networks which do not keep track of this information (e.g. Ethereum), this returns an empty array.
 
+---
+@{ref("js", "appOptIn")}@{ref("js", "stdlib.accountAppOptIn")}
+```js
+acc.appOptIn(contract) => Promise<void>
```

Comment:
I believe the type should be Contract and explicitly say it is the "info". I partially think it should be part of the ctc object actually

Reply:
I'm not sure I understand what you mean.  Do you mean you want the docs to say `appOptIn(contract: Contract)`?  Or you mean you would prefer this as `ctc.appOptIn(account)`?

Reply:
I had two ideas. The second sentence of mine and the third sentence of yours line up, but there's no need for the `account` argument, because a `ctc` knows its account. If you do that, then there will be no `contract` argument, so the first sentence isn't relevant here